src/vivarium_public_health/risks/distributions.py

```diff
@@ -213,7 +213,7 @@ def ppf(self, x: pd.Series) -> pd.Series:
 class DichotomousDistribution:
     def __init__(self, risk: str, exposure_data: pd.DataFrame):
         self.risk = risk
-        self.exposure_data = exposure_data.drop("cat2", axis=1)
+        self.exposure_data = exposure_data.drop(columns="cat2")
```

Comment:
Should cat2 be in a list?

Reply:
Good eye, but it looks like it handles single label or list-like

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.drop.html